File: src/main/java/com/getlose/guess/GameFor.java
package com.getlose.guess;

import java.util.Random;
import java.util.Scanner;

public class GameFor {

    public static void main(String[] args) {
        int secret = new Random().nextInt(10)+1;
        System.out.println("secret number is "+secret);
        Scanner scanner = new Scanner(System.in);
        for(int i = 1;i<50;i++){
            System.out.print("please key a number("+i+"/4):");
            int number = scanner.nextInt();
            if(secret>number){
                System.out.println("higher");
            }else if(secret < number){
                System.out.println("lower");
            }else{
                System.out.println("got it");
                break;
            }
        }
    }

}

File: src/main/kotlin/com/getlose/Game/GameFor.kt
package com.getlose.Game

import java.util.*

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    val secret = Random().nextInt(10)+1
    println(secret)
    for(i in 1..5){
//        print("please key a number:")
//        var number = scanner.nextInt()
//        println("第${i}次, ${number}")
//        if(number == -1)
//            break

        print("please key a number("+i+"/4):")
        var number = scanner.nextInt();
        if(secret>number){
            println("higher")
        }else if(secret < number){
            println("lower")
        }else{
            println("got it")
            break;
        }
    }

}